hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/service/OpenKeyCleanupService.java

```diff
@@ -265,7 +269,7 @@ private OMRequest createDeleteOpenKeysRequest(
 
     private OMResponse submitRequest(OMRequest omRequest) {
       try {
-        return OzoneManagerRatisUtils.submitRequest(ozoneManager, omRequest, clientId, runCount.get());
+        return OzoneManagerRatisUtils.submitRequest(ozoneManager, omRequest, clientId, callIdCount.incrementAndGet());
```

Comment:
With the callId used, runCount is of no use now.  runCount can be removed.

Reply:
`runCount` is used by multiple tests to get how many times service has run like in `testCleanupExpiredOpenKeys`.
So i used different name to distinguish it, What is your suggestion?

Reply:
Can testCleanupExpiredOpenKeys use callId to replace runCount?

Reply:
I have updated with callId, also in tests we are already waiting for SubmittedOpenKeyCount so I think we need not wait again for runCount.

Reply:
Then can we remove the getRunCount()?

Reply:
Removed it now, just was referenced in test logs.